plugins/frontend/backstage-plugin-github-pull-requests/src/components/useGithuRepositoryData.tsx

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 import { useApi, githubAuthApiRef } from '@backstage/core-plugin-api';
-import { Octokit } from '@octokit/rest';
 import { useAsync } from 'react-use';
 import { GithubRepositoryData } from '../types';
 import { useBaseUrl } from './useBaseUrl';
+import { GithubPullRequestsClient } from '../api';
 
 export const useGithuRepositoryData = (url: string) => {
```

Comment:
I know this isn’t changed not but could we fix the typo here and add ‘b’ in here so it is ‘useGithubRepositoryData’ 😁

Reply:
🤦🏿  ofc! thanks for catching it!